src/Microsoft.Framework.BufferedHtmlContent.Sources/BufferedHtmlContent.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Framework.Internal
     /// Enumerable object collection which knows how to write itself.
     /// </summary>
     [DebuggerDisplay("{DebuggerToString()}")]
-    internal class BufferedHtmlContent : IHtmlContent
+    internal class BufferedHtmlContent : IHtmlContentBuilder
```

Comment:
Bit strange that the builder interface inherits from "the what it's building" interface. But :ok: :